recipes/empyrical/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "empyrical" %}
+{% set version = "0.3.4" %}
+{% set sha256 = "f90a4c2c927510645a18f1c8f88658644cd45ad57d3eb187eb0fc31529ee27d7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - numpy
+    - pandas
+    - scipy
+    - pandas-datareader
+    - bottleneck
+
+test:
+  imports:
+    - empyrical
+    - empyrical.tests
+
+about:
+  home: https://github.com/quantopian/empyrical
+  license: Apache 2.0
+  license_family: Apache
+  # TODO - license_file: LICENSE
```

Comment:
Was the license not released with the code?

Reply:
No it wasn't...in fact, I am going to ask for it to be included in the next release on pypi. Can we go on without for the time being and update the conda-forge package with the license as soon as we push that fix on pypi?

Reply:
The Apache license requires that the license be in the package. You can package it manually https://conda-forge.org/docs/meta.html#packaging-the-license-manually